src/components/form/HdRange.vue

```diff
@@ -78,7 +78,12 @@ export default {
       };
     },
     stepsAmount() {
-      return 1 + (this.maxValue - this.minValue) / this.rangeStep;
+      const rangeSize = this.maxValue - this.minValue;
```

Comment:
You can make `rangeSize` a computed property by itself.

Reply:
👍